components/eventing-controller/controllers/subscriptionv2/eventmesh/test/utils.go

```diff
@@ -248,6 +271,15 @@ func startNewEventMeshMock() *reconcilertestingv1.BEBMock {
 	return emMock
 }
 
+func GenerateInvalidSubscriptionError(subName, errType string, path *field.Path) error {
+	webhookErr := "admission webhook \"vsubscription.kb.io\" denied the request: "
```

Comment:
vsubscription.kb.io ? I need some help understanding this :(

Reply:
this is autogenerated and comes from the validation webhook, and the `kb.io` I guess mean kubeBuilder.

Reply:
ah, thx for the explanation